test/unittest.hpp

```diff
@@ -29,6 +30,13 @@ struct StringMaker<boost::system::error_code> {
 };
 }
 
+inline std::vector<unsigned char> bytes_from_file(const std::string& path) {
+  std::ifstream ifs{path};
+  if (ifs.fail())
```

Comment:
Very minor thing but I prefer always to use braces even if it's a "one line conditional" like here. Not sure if clang-format could catch this.

Reply:
Ah yes, i tried to always use braces, but seems i missed them here. I will add them.
Seems clang-format 15 will introduce "InsertBraces" but it does not support it yet.

Reply:
Yes, it doesn't seem supported by clang-format yet although there seems to be a clang-tidy check for it.

It's not really that important but thanks for fixing it anyway,